strings/french.strings

```diff
@@ -58,64 +58,64 @@
 "Messages" = "Messages";
 
 /* MISSING TRANSLATION - model/contracts/group.js */
-"Admin credentials needed and not implemented yet." = "Admin credentials needed and not implemented yet.";
+"Admin credentials needed and not implemented yet." = "Un compte administrateur est requis mais n'est pas encore implementé.";
 
 /* MISSING TRANSLATION - model/contracts/group.js */
-"Cannot remove yourself." = "Cannot remove yourself.";
+"Cannot remove yourself." = "Vous ne pouvez pas vous exclure vous-même.";
 
 /* MISSING TRANSLATION - model/contracts/group.js */
-"Invalid associated proposal." = "Invalid associated proposal.";
+"Invalid associated proposal." = "La proposition associée est invalide.";
 
 /* MISSING TRANSLATION - model/contracts/group.js */
-"Not part of the group." = "Not part of the group.";
+"Not part of the group." = "Ne fait pas partie du groupe.";
 
 /* MISSING TRANSLATION - model/contracts/group.js */
-"Only the group creator can remove members." = "Only the group creator can remove members.";
+"Only the group creator can remove members." = "Seul le fondateur du groupe peut exclure des membres.";
 
 /* MISSING TRANSLATION - model/contracts/group.js */
-"There is an identical open proposal." = "There is an identical open proposal.";
+"There is an identical open proposal." = "Une proposition ouverte identique existe déjà.";
```

Comment:
Une proposition similaire a déjà été créée.

Reply:
Is it possible to keep "identical"? I don't understand why two similar but still not identical proposals could not be created.

Reply:
I think we should keep the original meaning of "identical" unless there is a strong reason not to

Reply:
I still prefer similaire in french to be honest, but I'm ok with this: Une proposition identique a déjà été créée.